src/components/PostInfo/PostInfo.jsx

```diff
@@ -2,0 +1,35 @@
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+import usersFromServer from '../../api/users.json';
+import commentsFromServer from '../../api/comments.json';
+
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const postComments = commentsFromServer.filter(
+    comment => comment.postId === post.id,
```

Comment:
The filtering logic for `postComments` is correct, but ensure that `commentsFromServer` is an array of comment objects with a `postId` field.